src/components/button/styles/CdrButton.module.scss

```diff
@@ -119,7 +119,7 @@
       @include cdr-button-large-mixin;
     }
 
-    &--fullWidth\@xs {
+    &--full-width\@xs {

```

Comment:
For some reason, this was camelCased here where every other style modifier is kebab-cased. Changing for consistency plus the JS generates a kebab'd class ... that was a fun bug to chase down.